README.md

```diff
@@ -266,6 +266,24 @@ Writing tests is very easy thanks to its syntax :
   });
 ```
 
+To run the Cypress tests, you need a local webserver. Run the following command in the same Python virtual environment as `udata`:
```

Comment:
I think the command lines should be in the opposite order : the cfg copy first, then the prepare and finally the serve

Reply:
Ah, yes, you're right, fixing this right away 🙏